tests/contrib/dd_trace_api/test_integration.py

```diff
@@ -0,0 +1,118 @@
+import sys
+from typing import Any
+
+import dd_trace_api
+import pytest
+
+from ddtrace._trace.span import Span as dd_span_class
+from ddtrace.contrib.internal.dd_trace_api.patch import patch
+from ddtrace.contrib.internal.dd_trace_api.patch import unpatch
+from tests.utils import TracerTestCase
+
+
+class DDTraceAPITestCase(TracerTestCase):
+    def setUp(self):
+        super(DDTraceAPITestCase, self).setUp()
+        patch(tracer=self.tracer)
+
+    def tearDown(self):
+        self.pop_spans()
+        super(DDTraceAPITestCase, self).tearDown()
+        unpatch()
+
+    def _assert_span_stub(self, stub: Any):
+        assert not isinstance(stub, dd_span_class), "Returned span object should be a stub"
+
+    def _assert_real_spans(self, count=1):
+        spans = self.pop_spans()
+        assert len(spans) == count
+        generated_span = spans[0]
```

Comment:
do you need to check them all if count > 1?

Reply:
i think checking one is enough for these tests